src/JuliusSweetland.OptiKey.Core/Observables/PointSources/PointServiceSource.cs

```diff
@@ -59,8 +58,8 @@ public IObservable<Timestamped<PointAndKeyValue>> Sequence
                             eh => pointGeneratingService.Point -= eh)
                         .Where(_ => State == RunningStates.Running)
                         .Select(ep => Settings.Default.KalmanFilterEnabled && pointGeneratingService.KalmanFilterSupported
-                            ? new Timestamped<Point>(new Point((int)kalmanFilterX.Update(ep.EventArgs.Value.X), (int)kalmanFilterY.Update(ep.EventArgs.Value.Y)), ep.EventArgs.Timestamp)
-                            : ep.EventArgs
+                            ? new Timestamped<Point>(kalmanFilter.Update(ep.EventArgs.Value, ep.EventArgs.Value.ToKeyValue(PointToKeyValueMap)), ep.EventArgs.Timestamp)
+                            : new Timestamped<Point>(gazeFilter.Update(ep.EventArgs.Value, ep.EventArgs.Value.ToKeyValue(PointToKeyValueMap)), ep.EventArgs.Timestamp)
```

Comment:
@AdamRoden This looks like the Kalman filter will be used if enabled, but if not then the GazeFilter will ALWAYS be used. Is that the intention? What do you think about the Management Console setting being something like "Additional smoothing" : Kalman, Adam's Filter, None?

Reply:
@JuliusSweetland For the final solution, we'll want to regroup some settings and decide what is best. As Kirsty said, we don't want to overwhelm users unnecessarily. So, we can offer all the options, but we'll need to make a call and put our best foot forward and denote the Default values if users start playing around.